Mathlib/Geometry/Euclidean/Sphere/Basic.lean

```diff
@@ -171,6 +177,12 @@ section NormedSpace
 
 variable [NormedAddCommGroup V] [NormedSpace ℝ V] [MetricSpace P] [NormedAddTorsor V P]
 
+lemma Sphere.exists_mem_iff [Nontrivial V] {s : Sphere P} : (∃ p, p ∈ s) ↔ 0 ≤ s.radius := by
+  refine ⟨fun ⟨p, hp⟩ ↦ radius_nonneg_of_mem hp, fun h ↦ ?_⟩
+  obtain ⟨v, hv⟩ := (NormedSpace.sphere_nonempty (x := (0 : V)) (r := s.radius)).2 h
```

Comment:
Why not 
```suggestion
  obtain ⟨v, hv⟩ := (NormedSpace.sphere_nonempty (x := (s.center : V)) (r := s.radius)).2 h
```

Reply:
I don't see how that would work, since `s.center` is in `P` not `V`.

Reply:
Ah, I think I was expecting a `NormedAddTorsor.sphere_nonempty` lemma about `Metric.sphere`. Could you add that one too, which should be defeq to this statement?

Reply:
Let's unblock this PR by leaving the extra lemma for a follow-up PR.

Reply:
> Where should such a lemma go? 

It could also just go right here in this file for now!